README.md

```
@@ -46,24 +64,23 @@ The template is a working application with a minimal setup. It contains:
 ## Setup
 
-Located in `./bin/init.sh`. Simply run and follow the explanation on how to
-execute it. This script will self-destruct when finished.
+Located in `./bin/init.py`. Run it and follow the prompts. The script
+self-destructs when finished. Requires Python 3.
 
 ### Enable All CI Pipelines
 
 - To enable all CI pipelines, set `REVIEWER_GITHUB_TOKEN` in the repository secrets both for `Action` and `Dependabot`
-- Create a new `Ruleset` for the `main` branch and for depednabot (example: [link](https://github.com/yonatankarp/spring-starter/settings/rules))
+- Create a new `Ruleset` for the `main` branch and for dependabot (example: [link](https://github.com/yonatankarp/spring-starter/settings/rules))
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)